js/editor-cm.js

```diff
@@ -14,6 +108,10 @@ function EditorCodeMirror(editorElement, settings) {
         'value': '',
         'autofocus': true,
         'matchBrackets': true,
+        'inputStyle': 'devToolsAccessibleTextArea',
+        // this is 25 days in milliseconds, we set the poll interval to be
+        // incredibly long so that a poll doesn't break our selection logic.
```

Comment:
perhaps comment/reiterate here why we don't need a regular poll - iiuc it's that it's designed to support IME input on other browsers and we only open in Chrome?

Reply:
Will do!